.github/workflows/main.yml

```diff
@@ -39,6 +39,7 @@ jobs:
       - run: cd client && yarn lint
       - run: cd client && yarn lint:test
       - run: cd client && yarn test
+      - run: "cd client && git rev-parse HEAD >git-sha"
```

Comment:
This won't generate the git-sha file on the server. For that we'll need to update the deploy tasks for multinet client

Reply:
Oh yeah of course.

Ok I'll look into making that change. We'll probably want to merge that change before this PR then, right?

Reply:
The app launches without the file, right? If so, it doesn't matter. If it won't build without the file then, yeah, it would need to be merged first

Reply:
Actually, we should merge the deploy branch first so that it doesn't get stuck without the file because the commit hash hasn't changed on this branch between the updates.

Reply:
Yeah that's what I was thinking. I am going to revert this change, though, because it's not necessary for CI. (The app builds properly whether or not the `git-sha` file is present.)